Lib9c/Action/IssueToken.cs

```diff
@@ -0,0 +1,98 @@
+using System.Collections.Generic;
+using System.Linq;
+using Bencodex.Types;
+using Lib9c;
+using Libplanet.Action;
+using Libplanet.Action.State;
+using Libplanet.Crypto;
+using Libplanet.Types.Assets;
+using Nekoyume.Action.Garages;
+using Nekoyume.Exceptions;
+using Nekoyume.Model.State;
+using Nekoyume.Module;
+using Nekoyume.TableData.Garages;
+
+namespace Nekoyume.Action
+{
+    [ActionType(TypeIdentifier)]
+    public class IssueToken : ActionBase
+    {
+        public const string TypeIdentifier = "issue_token";
+        public Address AvatarAddress;
+        public List<FungibleAssetValue> FungibleAssetValues;
+        public List<(int id, int count)> Items;
+
+        public IssueToken()
+        {
+        }
+
+        public override IValue PlainValue => Dictionary.Empty
+            .Add("type_id", TypeIdentifier)
+            .Add(
+                "values", Dictionary.Empty
+                    .Add("a", AvatarAddress.Serialize())
+                    .Add("f", new List(FungibleAssetValues.Select(f => f.Serialize())))
+                    .Add("i", new List(Items.Select(i => List.Empty.Add(i.id).Add(i.count))))
+            );
+
+        public override void LoadPlainValue(IValue plainValue)
+        {
+            var dict = (Dictionary)((Dictionary)plainValue)["values"];
+            AvatarAddress = dict["a"].ToAddress();
+            FungibleAssetValues = ((List)dict["f"]).ToList(f => f.ToFungibleAssetValue());
+            Items = new List<(int id, int count)>();
+            var list = (List)dict["i"];
+            foreach (var value in list)
+            {
+                var innerList = (List)value;
+                Items.Add(((Integer)innerList[0], (Integer)innerList[1]));
+            }
+        }
+
+        public override IWorld Execute(IActionContext context)
+        {
+            context.UseGas(1);
+            Addresses.CheckAvatarAddrIsContainedInAgent(context.Signer, AvatarAddress);
+            var state = context.PreviousState;
+            var sheet = state.GetSheet<LoadIntoMyGaragesCostSheet>();
+            var garageCost = sheet.GetGarageCost(FungibleAssetValues, Items);
+            state = state.TransferAsset(
+                context,
+                context.Signer,
+                Addresses.GarageWallet,
+                garageCost
+            );
+            foreach (var fungibleAssetValue in FungibleAssetValues)
+            {
+                var currency = fungibleAssetValue.Currency;
+                var wrappedCurrency = Currencies.GetWrappedCurrency(currency);
+                var recipient = GarageUtils.PickRecipient(context.Signer, AvatarAddress, currency);
```

Comment:
It may not be important. But I think it should use other word instead of `recipient` because nobody sends to the recipient and the recipient doesn't receive anything.